src/ode_solver/reduced_order_ode_solver.cpp

```diff
@@ -0,0 +1,299 @@
+#include "reduced_order_ode_solver.h"
+#include "dg/dg.h"
+#include "ode_solver_base.h"
+#include "linear_solver/linear_solver.h"
+#include "reduced_order/pod_basis_base.h"
+#include <deal.II/lac/trilinos_sparsity_pattern.h>
+#include <Epetra_Vector.h>
+#include <deal.II/lac/la_parallel_vector.h>
+#include <Epetra_LinearProblem.h>
+#include "Amesos_BaseSolver.h"
+#include <Amesos_Lapack.h>
+
+namespace PHiLiP {
+namespace ODE {
+
+template <int dim, typename real, typename MeshType>
+ReducedOrderODESolver<dim,real,MeshType>::ReducedOrderODESolver(std::shared_ptr< DGBase<dim, real, MeshType> > dg_input, std::shared_ptr<ProperOrthogonalDecomposition::PODBase<dim>> pod)
+        : ODESolverBase<dim,real,MeshType>(dg_input)
+        , pod(pod)
+{}
+
+template <int dim, typename real, typename MeshType>
+int ReducedOrderODESolver<dim,real,MeshType>::steady_state ()
+{
+    this->pcout << " Performing steady state analysis... " << std::endl;
+
+    if ((this->ode_param.ode_output) == Parameters::OutputEnum::verbose
+        && (this->current_iteration%this->ode_param.print_iteration_modulo) == 0
+        && dealii::Utilities::MPI::this_mpi_process(this->mpi_communicator) == 0 )
```

Comment:
As pcout is already initialized with mpi_rank ==0, maybe  `dealii::Utilities::MPI::this_mpi_process(this->mpi_communicator) == 0` in if statement can be removed.

Reply:
Fixed in most recent commit.